engine/sim/sc_sim.hpp

```diff
@@ -425,6 +425,8 @@ struct sim_t : private sc_thread_t
     timespan_t newfound_resolve_default_delay = 4_s;
     double newfound_resolve_delay_relstddev = 0.2;
     bool enable_rune_words = true;
+    /// Chance that the player will pickup Shredded Soul orb left by Ebonsoul Vise
+    double shredded_soul_pickup_chance = 1.0;
```

Comment:
There also needs to be a corresponding `add_option` call, otherwise this isn't doing much.